tests/test-amp-style-sanitizer.php

```diff
@@ -1136,14 +1136,20 @@ public function test_keyframe_sanitizer( $source, $expected = null, $expected_er
 	 * @returns array Stylesheet URL data.
 	 */
 	public function get_stylesheet_urls() {
+
+		// Make sure core-bundled themes are registered.
+		register_theme_directory( ABSPATH . 'wp-content/themes' );
```

Comment:
Maybe only do this conditionally if `ABSPATH . 'wp-content'` is actually different from `WP_CONTENT_DIR`?